staging/src/k8s.io/cloud-provider/app/controllermanager.go

```diff
@@ -95,7 +96,7 @@ the cloud specific control loops shipped with Kubernetes.`,
 				return err
 			}
 
-			c, err := s.Config(ControllerNames(controllerInitFuncConstructors), ControllersDisabledByDefault.List())
+			c, err := s.Config(ControllerNames(controllerInitFuncConstructors), ControllersDisabledByDefault.List(), []string{}, []string{})
```

Comment:
For readability it would be nice if constructed and passed in allWebhooks and disabledByDefaultWebhooks variables for the new parameters.